automated/health-checks/health-check_RevPi_Connect_001.yaml

```
@@ -0,0 +1,41 @@
+device_type: RevPi_Connect
+job_name: RevPi_Connect - Standard health check 001 
+priority: medium
+visibility: public
+
+timeouts:
+  job:
+    minutes: 5
+  action:
+    minutes: 1
+  connection:
+    minutes: 2
+    
+actions:
+- deploy:
+    timeout:
+      minutes: 4
+    to: ssh
+- boot:
+    method: ssh
+    connection: ssh
+    prompts: ["root@RevPi"]
```

From the LAVA documentation it is not immediately visible if this supports a Regex or not. If it does, I'd like it to match a more specific string.

```suggestion
    prompts: ["root@RevPi[0-9]{4}:"]
```

---

We were never satisfied with the prompt, so we used something general that worked.
Unfortunately for some reason the variables that can be set in the device dictionary do not work.
Last night I ran several tests using different options again, but only one has worked and that is when using different strings in the job.
The disadvantage of this, is that it would not work with RevPis that are not registered now... Obviously this is not a big problem, since the Job is executed with Lava-Web or lavacli.
We can talk about this later.

---

From Lava: 
`The full list of possible prompts must be specified by the test writer in the test job submission.`

[LAVA-Prompts](https://lava.revpi.kunbus.com/static/docs/v2/actions-boot.html?highlight=prompt#prompts)

---

Lava does not like the prompt either: [Here for example](https://lava.revpi.kunbus.com/scheduler/job/1563#L116)

---

If I understand correctly you can specify multiple prompts, i.e. `["root@RevPi[0-9]{4,}:", "root@RevPi:"]`.

> The full list of possible prompts must be specified by the test writer in the test job submission.

I'm not sure what this means... Definitions like "test writer" and "test job submission" are still very cloudy to me. The LAVA documentation so far hasn't done a great job at explaining these to someone unfamiliar with LAVA.